plugins/service_discovery/file.md

```diff
@@ -0,0 +1,134 @@
+# File Service Discovery Plugin
+
+The `file` serivce discovery plugin updates targets by reading local file.
+YAML and JSON are allowed as the file formats.
+
+## Example Configuration
+
+This is the exmple with out_forward.
+It udpates targets to send data by out_forward.
+
+```
+<source>
+  @type forward
+
+  <service_discovery>
+    @type file
+     path "/etc/fluentd/sd.yaml"
+  </service_discovery>
+</source>
+```
+
+Here is the example of target list file(`/etc/fluentd/sd.yaml`).
+
+```
+- 'host': 127.0.0.1
+  'port': 24224
+  'weight': 1
+  'name': server1
+- 'host': 127.0.0.1
+  'port': 24225
+  'weight': 1
+  'name': server2
+```
+
+## Parameters
+
+### @type
+
+The value must be `file`.
+
+### path
+
+| type   | default                 | version |
+|:-------|:------------------------|:--------|
+| string | `'/etc/fluent/sd.yaml'` | 0.17.0  |
```

Comment:
1.8.0, not 0.17.0. Other parameters are same.

Reply:
fixed in https://github.com/fluent/fluentd-docs-gitbook/pull/125

Reply:
in https://github.com/fluent/fluentd-docs-gitbook/pull/125

Reply:
ah, 1.8.0 is correct... fixed